src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  key: string | undefined
```

Comment:
Undefined means a variable has been declared but has not been assigned a value, while null is an assignment value set by a programmer. If you set it manually use null
```suggestion
  key: string | null
```